content/docs/architecture/quota-and-leases.md

```diff
@@ -48,22 +48,29 @@ in order to ensure that jobs are well-behaved against the cloud provider APIs.
 ## Adding a New Type Of Resource
 
 In order to add a new type of cloud quota to the system, changes to the `boskos` leasing server configuration are
-required. The configuration is checked into source control
-[here](https://github.com/openshift/release/blob/master/core-services/prow/02_config/_boskos.yaml).
-When adding a new type of quota, a new entry to the `resources` list is required, for example:
-
-`boskos` configuration:
-
-{{< highlight yaml >}}
-resources:
-- type: "my-new-quota-slice"
-  state: "free"
-  min-count: 10 # how many concurrent jobs can run against the cloud
-  max-count: 10 # set equal to `min-count`
+required. The configuration is checked into source control:
+
+* [generator](https://github.com/openshift/release/blob/master/core-services/prow/02_config/generate-boskos.py)
```

Comment:
Its still possible to manually add resources to the config or not?

Reply:
nope, because of the presubmit @stevekuznetsov requested for openshift/release#11752.